src/puripuly_heart/ui/controller.py

```
@@ -984,6 +1006,100 @@ async def start_discord_managed_auth_from_dialog(
+            self._show_short_message("qq_auth.error.retry")
+            return False
+
+        normalized_qq_identity = (qq_identity or "").strip()
+        normalized_credential = (credential or "").strip()
+        if not normalized_qq_identity or not normalized_credential:
+            logger.warning("[QQAuth] start_qq_managed_auth_from_dialog: empty input qq=%s credential=%s", bool(normalized_qq_identity), bool(normalized_credential))
+            self._show_short_message("qq_auth.error.invalid_input")
+            return False
+
+        self._discord_managed_auth_in_progress = True
+        self._set_managed_trial_pending_auth(True)
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)